lib/l10n/tr_nums.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:io';
+import 'package:intl/intl.dart';
+
+String trNums(num number) {
+  final formatter = NumberFormat('###.#', Platform.localeName);
+  return formatter.format(num.parse(format(number)));
+}
+
+String format(num n) {
+  return n.toStringAsFixed(n.truncateToDouble() == n ? 0 : 2);
+}
```

Comment:
Again, we don't need this

From [NumberFormat](https://api.flutter.dev/flutter/intl/NumberFormat-class.html) docs:
> `#` A single digit, omitted if the value is zero